src/graphics/programlib/programlib_phong.js

```diff
@@ -277,7 +277,7 @@ pc.programlib.phong = {
             } else {
                 code += chunks.specularAaNonePS;
             }
-            code += this._addMap("specular", options, chunks, uvOffset);
+            code += this._addMap(options.metalnessWorkflow? "metalness" : "specular", options, chunks, uvOffset);

```

Comment:
"internal boolean" is not intuitive.
metalnessWorkflow is internal, useMetalness is public.